ccw.core/src/java/ccw/util/osgi/ClojureOSGi.java

```diff
@@ -82,6 +81,10 @@ private static DynamicClassLoader getDynamicClassLoader(Bundle bundle, List<URL>
 	public static Object withBundle(Bundle aBundle, RunnableWithException aCode, List<URL> additionalURLs)
 			throws RuntimeException {
 
+		if (Thread.currentThread().isInterrupted() || CCWPlugin.canLoadCodeInBundle()) {

```

Comment:
no problem, already changed and pushed. Just asked to double check

2015-06-12 13:56 GMT+02:00 Andrea Richiardi <notifications@github.com>:

> In ccw.core/src/java/ccw/util/osgi/ClojureOSGi.java
> <https://github.com/laurentpetit/ccw/pull/782#discussion_r32309462>:
>
> > @@ -82,6 +81,10 @@ private static DynamicClassLoader getDynamicClassLoader(Bundle bundle, List<URL>
> >  	public static Object withBundle(Bundle aBundle, RunnableWithException aCode, List<URL> additionalURLs)
> >  			throws RuntimeException {
> >
> > +		if (Thread.currentThread().isInterrupted() || CCWPlugin.canLoadCodeInBundle()) {
>
> Yep sorry I am going to change!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laurentpetit/ccw/pull/782/files#r32309462>.
>



-- 
Laurent Petit